sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -69,6 +72,21 @@ AutoConfiguredOpenTelemetrySdkBuilder setConfig(ConfigProperties config) {
     return this;
   }
 
+  /**
+   * Adds a {@link BiFunction} to invoke the with the {@link SdkTracerProviderBuilder} to allow
+   * customization. The return value of the {@link BiFunction} will replace the passed-in argument.
+   *
+   * <p>Multiple calls will execute the customizers in order.
+   */
+  public AutoConfiguredOpenTelemetrySdkBuilder addTracerProviderCustomizer(
```

Comment:
Can you also add to this interface?

https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure-spi/src/main/java/io/opentelemetry/sdk/autoconfigure/spi/AutoConfigurationCustomizer.java

Reply:
Sure.